wcfsetup/install/files/acp/style/layout.scss

```diff
@@ -1,5 +1,7 @@
-var(--wcfAcpMenuWidth): 150px;
-var(--wcfAcpSubMenuWidth): 300px;
+html {
```

Comment:
Is there a difference between `:root` and `html`?

Reply:
Yes: `:root` is more specific than `html`:

```css
:root {
  --color: red;
}

html {
  --color: blue;
}

body: {
  color: var(--color); // red
}
```

Reply:
Yes, :root has a higher specifity than html (and it's the de facto standard for defining HTML variables). But I can't judge if there are reasons to define them in html here.

Reply:
My intention was to offer customization through `:root {}` without having to care about the selector order. In the long run this could be used to offer an easy way to overwrite CSS variables through the style editor without actually changing the generated values for `html {}`.

From a technical standpoint it doesn’t really matter which of these two selectors is actually used, because they will both target the same element. My approach has the upside of being able to make use of the difference in specificity, for better or for worse.